Userland/Libraries/LibTextCodec/Decoder.cpp

```diff
@@ -4,6 +4,8 @@
  * SPDX-License-Identifier: BSD-2-Clause
  */
 
+#include "ShiftJISConverter.h"
```

Comment:
Maybe it's time to split this out to multiple files?
if we continue like this, we'll have a many-hundred-thousand-line file we throw at the compiler, which is...not ideal

Reply:
Maybe it'd also be smart to move actual implementation to .cpp so that Shift-JIS header doesn't slow down the IDE when editing `Decoder.cpp`?

Reply:
that'd be a nice step in the right direction :)